Sitefinity CLI/PackageManagement/Implementations/NuGetV3DependencyParser.cs

```diff
@@ -80,9 +81,13 @@ private List<NuGetPackage> GetDependencies(IEnumerable<XElement> depElements)
             foreach (XElement depElement in depElements)
             {
                 string id = depElement.Attribute(Constants.IdAttribute)?.Value;
-                string version = depElement.Attribute(Constants.VersionAttribute)?.Value
+
+                string versionFromXml = depElement.Attribute(Constants.VersionAttribute)?.Value
                     .Trim(new char[] { '[', '(', ')', ']' });
 
+                string[] dependencyVersions = this.ParseVersionString(versionFromXml);
```

Comment:
Above you have null conditional operator which will skip getting the Value in that case. You have to handle the probability versionFromXml to be null later as well